### ISSUE TITLE
ADA - Role with implied hidden content has keyboard focus

### ISSUE THREAD

**Issue by rbruhn** [AustinMastLab/WeDigBio-D7#2]
Some elements have a role that implies their internal structure should be hidden from assistive technologies. In other words, their sub-elements have a presentational role.

If a sub-element has keyboard focus, it should be moved outside of the presentation role.

<img width="863" height="209" alt="Image" src="https://github.com/user-attachments/assets/f4822f19-fdeb-4ee8-afba-ce0eaca9904b" />
